install/migrations/update_10.0.5_to_10.0.6/configs.php

```diff
@@ -40,5 +40,7 @@
 $migration->displayMessage('Add new configurations / user preferences');
 $migration->addConfig([
     'timeline_action_btn_layout'   => 0,
+    'timeline_relative_date'   => 0,
 ]);
 $migration->addField('glpi_users', 'timeline_action_btn_layout', 'tinyint DEFAULT 0');
+$migration->addField('glpi_users', 'timeline_relative_date', 'tinyint DEFAULT 0');
```

Comment:
Config name/value is ambiguous. One would expect `0` value to be considered as `false` (= do NOT display relative dates) and `1` to be considered as `true` (= do display relative dates), but this is not the case.

Config should be renamed `timeline_date_format` to prevent any ambiguity.

Reply:
option renamed